src/App.css

```diff
@@ -0,0 +1,186 @@
+.country-list {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+}
+
+.country-item {
+  width: 48%; /* Each item takes up 48% width to fit two in a row */
+  margin-bottom: 20px;
+}
+
+.country-card {
+  display: flex;
+  text-decoration: none;
+  padding: 10px;
+  border-radius: 4px;
+  box-shadow: 0 5px 7px rgb(6 5 5 / 10%);
+  text-align: center;
+  color: #fff6f6;
+  flex-direction: column;
+  height: 293px;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.forward-icon {
+  margin-left: 5px;
+}
+
+.search-input {
+  text-align: center;
+}
+
+.search-input::placeholder {
+  text-align: center;
+}
+
+.country-card:hover {
+  box-shadow: 0 5px 7px rgb(6 5 5 / 15%);
+  height: 295px;
+  background: #f95593;
+  width: 413px;
+}
+
+.country-flag {
+  height: 250px;
+  width: auto;
+  max-width: 100%;
+  margin-bottom: 10px;
+}
```

Comment:
- [ ] I kindly suggest adjusting the flag size behavior upon hovering over a country. Currently, the flag size switches between medium on desktop screens and large on mobile screens. It would be appreciated if you could make the necessary adjustments accordingly.